src/udp_wrap.cc

```diff
@@ -219,6 +221,35 @@ void UDPWrap::Bind6(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UDPWrap::BufferSize(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK(args[0]->IsUint32());
+  CHECK(args[1]->IsUint32());
+  int size = static_cast<int>(args[0].As<Uint32>()->Value());
```

Comment:
Would you recommend just passing down a signed int from the JS layer? The libuv layer accepts signed int: https://github.com/nodejs/node/blob/master/deps/uv/src/uv-common.c#L445

I'm actually thinking if we should just let the OS deal with values if they are outside a range it supports. Linux looks to choose a max or min if input is outside a range it supports and not throw an error:
https://github.com/torvalds/linux/blob/8b4822de59d5d9919b9b045183a36c673ce20b73/net/core/sock.c#L752

I cannot find any documentation on what the behavior is on Windows.